src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -169,6 +171,43 @@ class TransferBalancePage extends React.Component {
     }
 
     render() {
+        if (this.props.walletTransfer.shouldShowTransferSuccess) {
+            return (
+                <ScreenWrapper>
+                    <KeyboardAvoidingView style={[styles.flex1]} behavior="height">
+                        <HeaderWithCloseButton
+                            title={this.props.translate('common.transferBalance')}
+                            onCloseButtonPress={PaymentMethods.dismissSuccessfulTransferBalancePage}
+                        />
+                        <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                            <Icon
+                                src={Illustrations.TadaYellow}
```

Comment:
I left this in the other issue, but here is a blue version in case you want to switch it out: [tada--blue.svg.zip](https://github.com/Expensify/App/files/9075044/tada--blue.svg.zip)

Reply:
Thanks, Updated!

![Screen Shot 2022-07-11 at 1 28 08 PM](https://user-images.githubusercontent.com/12268372/178235220-ec067568-01e6-492a-88af-e1bcec9b409a.png)

Reply:
Looks good!